### ISSUE TITLE
can logged data be summarized or reorganized?

### ISSUE THREAD

**Issue by ahillio** [Webskillet/rules_api#1]
The information I most want to see in the logs is "resultCode" and "errors".  This information is at the bottom of the log message.  It would be great to have them visible from `drush ws` but the end of the message gets clipped off.
![selection_288](https://cloud.githubusercontent.com/assets/5461756/26252447/3852942e-3c7e-11e7-8e79-b08208be6622.jpg)
![desktop 1_287](https://cloud.githubusercontent.com/assets/5461756/26252462/3ff7994a-3c7e-11e7-8cc8-cee6828771ba.jpg)

**Comment by jkissam** [comment]
yes - if you want to patch/branch it quickly at the code level, you could:

a) remove all the array keys except 'data_returned' from line 152 in rules_api.rules.inc (this would still log all of the returned data, but you wouldn't get any of the call)

or

b) replace lines 86-94 with

`$log= isset($return['data']) ? $return['data'] : array();`

and then define `$fields` in line 152 as `array('resultCode','errors');`

**Comment by jkissam** [comment]
longer-term, I could add controls for this in the UI

**Comment by ahillio** [comment]
Oh cool I'll give that a shot next time I'm testing a rule.
